Levels/SpawningPerfTest/_savebackup/2022-01-20 [09.42.00]/tags.txt

```diff
@@ -0,0 +1,12 @@
+0,0,0,0,0,0
```

Comment:
The save backup. We should git ignore these…

Reply:
Good catch, removed.